superset/models/dashboard.py

```diff
@@ -294,7 +293,7 @@ def update_thumbnail(self) -> None:
 
     @debounce(0.1)
     def clear_cache(self) -> None:
-        cache.delete_memoized(self.full_data)
```

Comment:
The original code only hash version related to the function:  https://github.com/sh4nks/flask-caching/blob/v1.9.0/flask_caching/__init__.py#L1051-L1052